Doc/using/cmdline.rst

```diff
@@ -404,10 +404,15 @@ Miscellaneous options
 
    Warn about Python 3.x possible incompatibilities by emitting a
    :exc:`DeprecationWarning` for features that are removed or significantly
-   changed in Python 3.
+   changed in Python 3 and can't be detected using static code analysis.
 
    .. versionadded:: 2.6
 
+   .. note::
```

Comment:
I don't think a note directive is needed here. You can say something like

```rst
See :doc:`howto/pyporting` for more details.
```

I haven't checked whether :doc:\`howto/pyporting\` works or not so please build the docs locally and test it.